packages/backend-common/src/sqs.ts

```diff
@@ -86,13 +86,14 @@ export const generateOutputSignedUrlAndSendMessage = async (
 		originalFilename,
 		outputBucketUrls: signedUrls,
 	};
-	return await sendMessage(client, queueUrl, JSON.stringify(job));
+	return await sendMessage(client, queueUrl, JSON.stringify(job), id);
 };
 
 const sendMessage = async (
 	client: SQSClient,
 	queueUrl: string,
 	messageBody: string,
+	id: string,
```

Comment:
on main, is this being used at the moment? 🤔

Reply:
We just checked with Zeke, and you may need a rebase from main. Not sure why github is not showing the diff with the latest main 🤔

Reply:
Thanks for this review, I have fixed it now. Apologies for these rushed PRs - should have properly tested to start with but got overexcited.